src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,32 @@
+"use client";
+
+import { useState } from "react";
+
+import { Icon } from "@/components/Icon/Icon";
+
+type PopoverProps = {
+  children: React.ReactNode;
+};
+export const Popover: React.FC<PopoverProps> = (props) => {
+  const handlePopOverOpen = () => {
+    return setIsNavbarOpen(!isNavbarOpen);
+  };
+  const [isNavbarOpen, setIsNavbarOpen] = useState(false);
```

Comment:
useStateで定義した値をSet関数で変更するので逆の方がいいのと`handlePopOverOpen`自体は何も返さない関数なのでreturn いらないです。
命名もPopOverをOpenなのかNavBarをOpenなのかどちらかに揃えた方が良いかと思います。
```suggestion
  const [isNavbarOpen, setIsNavbarOpen] = useState(false);
  const handlePopOverOpen = () => setIsNavbarOpen(!isNavbarOpen);
```